libfastjson.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: libfastjson
+  version: 1.2304.0
+  epoch: 0
+  description: a fast json library for C
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3a8402c1de7c7747c95229db26d8d32fb85a7a52
+      repository: https://github.com/rsyslog/libfastjson
+      tag: v${{package.version}}
+
+  - runs: |
+      libtoolize --copy
+      aclocal -I m4 --install
+      autoheader
+      autoconf
+      automake --foreign --add-missing --force-missing --copy
+      ./configure --prefix=/usr
+      make -j $(nproc)
+      make install
```

Comment:
```suggestion
      make DESTDIR="${{targets.destdir}}" install
```

Reply:
Yes!  Thanks!!!  I was trying to figure that out just now...